<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QWidget</class>
 <widget class="QWidget" name="QWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>721</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>80</y>
     <width>658</width>
     <height>536</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <property name="spacing">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>                 </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Minimize">
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Maximize">
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Cross">
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QPushButton" name="Menu">
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Home">
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Settings">
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,0">
        <property name="spacing">
         <number>10</number>
        </property>
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <item alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="label">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>100</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="resource.qrc">:/Icons/Screenshot 2024-04-22 at 9.53.55 AM.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <property name="openExternalLinks">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>10</number>
          </property>
          <property name="leftMargin">
           <number>10</number>
          </property>
          <property name="rightMargin">
           <number>10</number>
          </property>
          <property name="bottomMargin">
           <number>10</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>From Date</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDateEdit" name="FromDate"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>To Date</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDateEdit" name="ToDate"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Hour</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="Hour">
              <property name="maximumSize">
               <size>
                <width>200</width>
                <height>100</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>* if u want for all the hours , leave the hours column blank</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QCheckBox" name="SelectedButton">
              <property name="text">
               <string>Selected Deals</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="PendingButton">
              <property name="text">
               <string>Pending Deals</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="NonSelectedButton">
              <property name="text">
               <string>Non-Selected Deals</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="table"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
